sections/semantics-sections.include

```diff
@@ -1255,10 +1255,10 @@
   The <code>h1</code>–<code>h6</code> elements are headings.
 
   The first element of <a>heading content</a> in an element of <a>sectioning content</a>
-  <a>represents</a> the heading for that explicit section. Subsequent headings of equal or higher <a>rank</a>
-  start new (implied) subsections that are part of the previous section's parent section. Subsequent
-  headings of lower <a>rank</a> start new implied subsections that are part of the previous one.
-  In both cases, the element <a>represents</a> the heading of the implied section.
+  <a>represents</a> the heading for that (explicit) section. Subsequent headings of equal or higher
+  <a>rank</a> start new (implied) subsections that are part of the previous section's parent section.
+  Subsequent headings of lower <a>rank</a> start new (implied) subsections that are part of the previous
+  one. In both cases, the element <a>represents</a> the heading of the implied section.
```

Comment:
I prefer to avoid parentheses where possible.

Also, here, s/subsections that are part of the previous one/subsections of the current section/